os/drivers/ai-soc/ndp120/src/ndp120_api.c

```diff
@@ -984,7 +1009,7 @@ ndp120_app_device_health_check(void *d)
 			printf("Error: %d in check_firmware_aliveness\n", s);
 			goto out;
 		}
-		pm_sleep(5000);
+		pm_sleep(3000);
```

Comment:
What is this magic number about? Can we have config for this?

Reply:
Done